pom.xml

```diff
@@ -61,6 +61,7 @@
                 <groupId>com.vaadin</groupId>
                 <artifactId>flow</artifactId>
                 <version>${vaadin.flow.version}</version>
+                <type>pom</type>
```

Comment:
If the intend is to import `flow` as a bom, then it's missing the scope import. Importing a pom as a regular dependency is very unusual. Can you please share a bit more context about this?

Reply:
I see, thanks for the feedback. Agreed that importing a `pom` artifact like that is a smell and should be fixed in its own right.